projects/consensus-service-typescript/src/raft-consensus.ts

```diff
@@ -93,14 +99,6 @@ export default class RaftConsensus {
 
       const txData = msgData.msg;
 
-      // Verify the sending node's signature, in order to avoid internal spam or miscommunication.
```

Comment:
Can you restore and comment it and/or add `TODO: `?

Reply:
I added a task on this issue. I'm not sure at all the verification belongs here and therefore don't see any point in restoring this comment